diesel_migrations/migrations_internals/src/connection.rs

```diff
@@ -26,7 +25,7 @@ where
     T: Connection,
     String: FromSql<VarChar, T::Backend>,
     // FIXME: HRTB is preventing projecting on any associated types here
-    for<'a> InsertStatement<__diesel_schema_migrations, Grouped<ColumnInsertValue<version, &'a Bound<VarChar, &'a str>>>>: ExecuteDsl<T>,
+    for<'a> InsertStatement<__diesel_schema_migrations, ValuesClause<ColumnInsertValue<version, &'a Bound<VarChar, &'a str>>, __diesel_schema_migrations>>: ExecuteDsl<T>,
```

Comment:
Shouldn't this not better use `<InsertType as Insertable<__diesel_schema_migrations>>::Values`?

Reply:
See the comment on the line above